src/main/java/com/lgtm/easymoney/payload/FeedActivityRsp.java

```diff
@@ -0,0 +1,24 @@
+package com.lgtm.easymoney.payload;
+
+import com.lgtm.easymoney.enums.Category;
+import com.lgtm.easymoney.enums.TransactionStatus;
+import java.math.BigDecimal;
+import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * transaction response.
+ */
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class FeedActivityRsp {
+  private Long fromUid;
+  private Long toUid;
+  private String amount;
```

Comment:
to reviewers: need string to hide friend's activities(maybe in next pr)

Reply:
I think friend's activities should have amount = null